addons/base_automation/views/base_automation_views.xml

```diff
@@ -233,46 +229,41 @@
+                                    <field name="model_id" invisible="context.get('default_model_id')"/>
+                                    <div class="d-flex align-items-center gap-1" invisible="trigger in ['on_time', 'on_time_created', 'on_time_updated']">
+                                        <field name="trigger" />
+                                        <field name="on_change_field_ids" invisible="trigger != 'on_change'" class="my-1" />
+                                        <field name="trg_selection_field_id" invisible="trigger not in ['on_state_set', 'on_priority_set']" class="o_tag o_tag_color_0 rounded-pill p-1 px-2" />
+                                        <field name="trg_field_ref" invisible="trigger not in ['on_stage_set', 'on_tag_set']" class="o_tag o_tag_color_0 rounded-pill p-1 px-2" />
+                                        <field name="trigger_field_ids" invisible="trigger not in ['on_create_or_write']" class="my-1" />
                                     </div>
-                                    <div class="d-none flex-grow-1 align-items-center gap-3 o_automation_actions" data-name="more-info">
-                                        <i class="fa fa-2x fa-arrow-right text-primary" title="Actions" />
-                                        <field name="action_server_ids" widget="base_automation_actions_one2many" class="align-self-center w-100 me-md-3" />
+                                    <div class="d-flex align-items-center gap-1" invisible="trigger != 'on_time'">
+                                        <field name="trg_date_range"/>
+                                        <field name="trg_date_range_type" class="text-lowercase text-nowrap" />
+                                        <span class="flex-shrink-0">
+                                            based on <field name="trg_date_id" invisible="trigger != 'on_time'"  class="o_tag o_tag_color_0 rounded-pill p-1 px-2" />
+                                        </span>
                                     </div>
+                                    <div class="d-flex align-items-center gap-1" invisible="trigger not in ['on_time_created', 'on_time_updated']">
+                                        <field name="trg_date_range"/>
+                                        <field name="trg_date_range_type" class="text-lowercase" />
+                                        <field name="trigger" class="text-lowercase" />
+                                        <field name="trg_date_id" invisible="trigger != 'on_time'"  class="o_tag o_tag_color_0 rounded-pill p-1 px-2" />
+                                    </div>
+                                    <field name="trg_date_calendar_id" invisible="not trg_date_id or trg_date_range_type != 'day'" />
                                 </div>
                             </div>
+                            <div class="d-none flex-grow-1 align-items-center gap-3 o_automation_actions" data-name="more-info">
```

Comment:
remove class (and style) o_automation_actions, deal with d-xs/md-none class instead

Reply:
Done. I used the data-name attribute in the tour instead of the o_automation_actions class.